docs/hello.rst

```diff
@@ -113,25 +114,42 @@ size ``(640, 320)`` and element data type ``ti.f32`` (i.e. ``float`` in C).
 Functions and kernels
 ---------------------
 
-Computation resides in Taichi **kernels**. Kernel arguments must be type-hinted.
-The language used in Taichi kernels and functions looks exactly like Python, yet the Taichi frontend compiler converts it
-into a language that is **compiled, statically-typed, lexically-scoped, parallel and differentiable**.
+Computation resides in Taichi **kernels** and Taichi **functions**.
+
+Taichi **kernels** are defined with the decorator ``@ti.kernel``.
+They can be called from Python to perform computation.
+Kernel arguments must be type-hinted (if any).
+
+Taichi **functions** are defined with the decorator ``@ti.func``.
+They can be called by Taichi kernels or other Taichi functions.
 
-Taichi **functions**, which can be called by Taichi kernels and other Taichi functions, should be defined with the keyword ``ti.func``.
+See :ref:`syntax` for more details about Taichi kernels and functions.
+
+The language used in Taichi kernels and functions looks exactly like Python, yet the Taichi frontend compiler converts it into a language that is **compiled, statically-typed, lexically-scoped, parallel and differentiable**.
 
 .. note::
 
-  **Taichi-scopes v.s. Python-scopes**: everything decorated with ``ti.kernel`` and ``ti.func`` is in Taichi-scope, which will be compiled by the Taichi compiler.
-  Everything else is in Python-scopes. They are simply Python code.
+  **Taichi-scopes v.s. Python-scopes**:
+
+  Everything decorated with ``@ti.kernel`` and ``@ti.func`` is in Taichi-scope
+  and hence will be compiled by the Taichi compiler.
+
+  Everything else is in Python-scopes. They are simply Python native code.
```

Comment:
Shall we unify the format for `Python-scope` and `Taichi-scope`? If one is using plural, the other one should too.

Reply:
```suggestion
  Everything decorated with ``@ti.kernel`` and ``@ti.func`` is in Taichi-scope
  and hence will be compiled by the Taichi compiler.

  Everything else is in Python-scope. They are simply Python native code.
```
Yes, I believe non-plural form is easier to read.